pyEpiabm/pyEpiabm/routine/toy_population_config.py

```diff
@@ -168,12 +168,29 @@ def assign_cell_locations(population: Population, method: str = 'random'):
             if method == "random":
                 for cell in population.cells:
                     cell.set_location(tuple(np.random.rand(2)))
+                    for microcell in cell.microcells:
+                        inter_dist = [0]
+                        cell_dist = 100
+                        while min(inter_dist) < cell_dist:
```

Comment:
This is a little ugly given we never use these default values - a do/while loop might be more suitable here?

Reply:
Minor reworking, but lack of do while in python does make this ugly!